chart/values.yaml

```diff
@@ -10,6 +10,9 @@ image:
   # Overrides the image tag whose default is the chart appVersion.
   tag: ""
 
+subscription:
+  pubsubTopic: "sh.keptn.event.monitoring.configure,sh.keptn.event.configure-monitoring.triggered,sh.keptn.event.get-sli.triggered" # Sets the events the service subscribes to
```

Comment:
we should remove `sh.keptn.event.monitoring.configure`

Reply:
Since the implementation in the core repository will not be changed, the `sh.keptn.event.monitoring.configure` event needs to stay. I removed the `sh.keptn.event.configure-monitoring.triggered` instead.